### PR TITLE
Add ability to sort asc/desc through headers

### PR DESCRIPTION
**Pull request by eanlain**

### CODE REVIEW

File: app/helpers/storytime_admin/application_helper.rb [hunk 1]
```diff
@@ -24,5 +24,37 @@ def respond_to?(method)
         super
       end
     end
+
+    def setup_column_sort(attribute)
+      if params[:sort]
```
Comment (dvanderbeek) [inline app/helpers/storytime_admin/application_helper.rb]: seems like these conditionals could be simplified quite a bit. The outcomes are always either "asc" or "desc"...we should be able to just have 1 if statement

**Comment by eanlain** [comment]
@dvanderbeek I just made an update to the conditional in the setup_column_sort method in the application helper. Better?

**Comment by dvanderbeek** [comment]
that works. I think the code from http://railscasts.com/episodes/240-search-sort-paginate-with-ajax is a bit simpler than this, but we can refactor it later. Any tests for this?

**Comment by eanlain** [comment]
@dvanderbeek refactored and tests added! Can you check it out?

**Comment by dvanderbeek** [comment]
Looks awesome

**Pull request merged**